h2o-r/tests/runitUtils/utilsR.R

```diff
@@ -936,12 +936,20 @@ compareFrames <- function(frame1, frame2, prob=0.5, tolerance=1e-6) {
   for (colInd in range(1, ncol(frame1))) {
     temp1=as.numeric(frame1[,colInd])
     temp2=as.numeric(frame2[,colInd])
+    notNumericCols = !(h2o.isnumeric(frame1[,colInd]) && h2o.isnumeric(frame2[,colInd]))
+    if (notNumericCols) {
```

Comment:
move the above assignment into else branch

Reply:
Done.